spec/adapters/alma_adapter_spec.rb

```diff
@@ -233,18 +155,6 @@
             }
           )
           .to_return(status: 200, body: "{}", headers: { "content-Type" => "application/json" })
-
```

Comment:
Can the stub above also be removed? It was added in https://github.com/pulibrary/marc_liberation/pull/1052

Reply:
Good catch. Removed.